content/cloud/noops-sql-db.md

```diff
@@ -69,34 +67,41 @@ name = "c3"
 sqlite_databases = ["default"]
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+The above configuration will last the lifespan of the application.
 
-```bash
-$ spin cloud deploy
-Deploying app ..
-Creating database inspirational-pig (default) for app "quickstart"
-```
+### Preparing Fermyon Cloud Schema
 
-You can check to see your database by using the `spin cloud sqlite` command:
+Bootstrapping your Cloud SQLite database schema is done via the `spin cloud sqlite execute` subcommand. Your NoOps SQLite application on localhost will have already assigned a name for your database. All we need to do is obtain the name by typing the following command (in your local NoOps SQLite application's directory):
 
->> Fermyon Cloud will automatically generate a random name for your database.
+<!-- @selectiveCpy -->
 
 ```bash
 $ spin cloud sqlite list
-inspirational-pig (default) 
+Databases (1)
+inspirational-pig (default)
+```
+
+Once we have this name (in this above example, we can see our database called `inspirational-pig`), we use that `inspirational-pig` as part of our `spin cloud sqlite execute` subcommand to create our schema for our Fermyon Cloud application:
+
+```bash
+$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
 ```
 
+As mentioned above, Fermyon Cloud only supports databases named “default” at this time. Once you add the `sqlite_databases = ["default"]` line to your local Spin application's manifest, Spin internally associates that `default` database with the random name Spin autogenerates, i.e. `inspirational-pig`. The name is not arbitrary and must be used explicitly when using the `spin cloud sqlite execute` subcommand (as shown above). This will allow you to develop your application, continually build locally, and deploy to Fermyon Cloud, knowing that the two instances (localhost and Cloud) share the same schema.
```

Comment:
I am confused how the beginning of this paragraph leads to the ending point of the fact that the "default" name in a `spin.toml` helps you know "that the two instances (localhost and Cloud) share the same schema."

Reply:
Thanks so much @itowlson and @kate-goldenring 
Perhaps the approach for now (after reading the latest `spin cloud link` meeting notes etc.) is that less is more.
The docs can perhaps simply say that:
- Fermyon Cloud only supports databases named “default”
- A user can obtain the name of the database that Spin created via `spin cloud sqlite list`
- The `spin cloud sqlite execute` takes the database name as an argument.
I see some comments in the file view; I will head over and make some updates and ping in here when finished. Would be brilliant to finalise this. Thanks again for your help.

Reply:
Thanks this has been reworded now.